source/mainnet/smart-contracts/tutorials/sponsoredTransactions/walletConnect.rst

```diff
@@ -5,6 +5,5 @@
 WalletConnect
 =============
 
-The |mw-gen2| are in the process of being updated to support signing byte messages with the ``signMessage`` request.
-This tutorial page will be available when the relevant version of |mw-gen2| is released.
-
+The |mw-gen2| is in the process of being updated to support signing byte messages with the ``signMessage`` request.
+This tutorial page will be available when the relevant version of the |mw-gen2| is released.
```

Comment:
Are there any actual plans to do this? I'm wondering if we should just entirely delete this page - or dare I say, even this whole tutorial. It's kind of half-finished as is.

Reply:
We could delete this page since the `WalletConnect` connection has not been implemented in the `sponsoredTransaction` examples yet. But deleting the whole tutorial? Is the `sponsoredTransaction` not the only tutorial that explains how signatures can be created and then verified in smart contracts?

Reply:
Probably -  I'll delete the page and leave the first part.